src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -65,21 +68,19 @@ const PlanDefinitionList = (props: PlanListProps) => {
   /** async function to load the data */
   async function loadData() {
     try {
-      setLoading(plans.length < 1); // only set loading when there are no plans
       const planObjects = await apiService.list();
       fetchPlans(planObjects);
     } catch (e) {
-      // do something with the error?
-    } finally {
-      setLoading(false);
+      growl(e.message, { type: toast.TYPE.ERROR });
     }
   }
 
   useEffect(() => {
     loadData();
+    return () => controller.abort();
   }, []);
 
-  if (loading === true) {
+  if (plans.length < 1) {
     return <Loading />;
   }
```

Comment:
Same as above

Reply:
`loading` state variable is computed from `plans.length` here too